libraries/adaptive-expressions/src/builtinFunctions/xml.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Expression } from '../expression';
+import { EvaluateExpressionDelegate, ExpressionEvaluator } from '../expressionEvaluator';
+import { ExpressionType } from '../expressionType';
+import { FunctionUtils } from '../functionUtils';
+import { ReturnType } from '../returnType';
+
+/**
+ * Return the newline string according to the environment.
+ */
+export class XML extends ExpressionEvaluator {
+    /**
+     * Initializes a new instance of the [EOL](xref:adaptive-expressions.EOL) class.
+     */
+    public constructor() {
+        super(ExpressionType.XML, XML.evaluator(), ReturnType.String, FunctionUtils.validateUnary);
+    }
+
+    /**
+     * @private
```

Comment:
Our JSDoc lint rule will only require comments on public or protected methods, not private ones. This comment block should be omitted.

Reply:
removed